blockchain/dkg/dkgtasks/dkg_task_mock.go

```diff
@@ -1,72 +0,0 @@
-package dkgtasks
```

Comment:
Are the new mocks being able to test the ethdkg and snapshot golang tasks?

Reply:
Yes!